### PR TITLE
added second measure and html with styling to viz tooltip.

### PR DESCRIPTION
**Pull request by jasongotlieb**

### CODE REVIEW

File: src/bubble_chart/bubble_chart.js [hunk 1]
```diff
@@ -205,10 +206,10 @@ class BubbleChart extends Component {
     node
       .on("mousemove", function (d) {
         d3.select("#chart").append("div").attr("id", "tooltip");
-
         let tooltip_html = "";
         tooltip_html += "<div><span>" + d.data.itemName + "<br/></span>";
-        tooltip_html += "<span>   " + d.data.rendered + "</span></div>";
+        tooltip_html += "<span>   " + (d.data.html ? DOMPurify.sanitize(d.data.html) : d.data.rendered) + "<br/></span>";
+        tooltip_html += "<span>   " + (d.data.secondMeasureHtml ? DOMPurify.sanitize(d.data.secondMeasureHtml) : d.data.secondmeasureRendered) + "</span></div>";
```
Comment (bryans99) [inline src/bubble_chart/bubble_chart.js:212]: `secondmeasureRendered`

Lower case 'm' for measure?

`secondMeasureHtml` has upper case 'M'

**Review by bryans99 — COMMENTED** [review]
See question

File: src/bubble_chart/bubble_chart.js (hunk 1 above)
Comment (jasongotlieb) [reply src/bubble_chart/bubble_chart.js]: Ok. Updated this in both places. Thank you!

**Comment by jasongotlieb** [comment]
Here's a screenshot of the second measure showing up in the toolti
Screen Shot 2022-05-09 at 12 25 18 PM
Screen Shot 2022-05-09 at 12 18 04 PM
p

**Pull request merged**